modules/cas_cache/ocf_env.h

```diff
@@ -40,14 +40,24 @@ static inline void env_free(const void *ptr)
 	kfree(ptr);
 }
 
+static inline void *env_vmalloc_flags(size_t size, int flags)
+{
+	return __vmalloc(size, flags | __GFP_HIGHMEM, PAGE_KERNEL);
```

Comment:
If any kind of kernel flags are going to be passed from OCF code, appropriate ENV_ macros should be provided.

Reply:
flags are already defined ocf_env.h, line 19